README.adoc

```diff
@@ -3,15 +3,18 @@
 // 2. Update the issues link in Contributing section in the readme file
 // 3. Update the discussion link in config.yml file in .github/ISSUE_TEMPLATE directory
 
-= repo-template
+= CNF-01: Java Configuration Library
 
 // Add a short description of your project. Tell what your project does and what it's used for.
 
-This is a template repository for Teragrep organization.
+CNF-01 is a library that provides immutable configuration for Java projects. Immutability is achieved by converting the configuration source into an immutable Map. The resulting configuration Map can't be altered even with modifying the underlying source.
 
 == Features
 
 // List your project's features
+- Provides immutable configurations for:
+. Configuration files / path properties (`PathConfiguration`)
```

Comment:
There should be more than one listing item in lists. Add more or combine it with the previous one like this: "Provides immutable configurations for `PathConfiguration`" or something like that.